src/python/pants/backend/codegen/tasks/antlr_gen.py

```diff
@@ -1,5 +1,5 @@
 # coding=utf-8
-# Copyright 2014 Pants project contributors (see CONTRIBUTORS.md).
+# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Only change the copyright to 2015 if it's a new file you just added, otherwise leave it as it was.